Makefile-rpm-ostree.am

```diff
@@ -57,3 +57,10 @@ rpm_ostree_LDADD = $(PKGDEP_RPMOSTREE_LIBS) librpmostreepriv.la librpmostree-1.l
 
 privdatadir=$(pkglibdir)
 privdata_DATA = src/app/rpm-ostree-0-integration.conf
+
+install-bin-hook:
+	mv $(DESTDIR)$(bindir)/rpm-ostree $(DESTDIR)$(bindir)/$(primaryname)
+	ln -sf $(primaryname) $(DESTDIR)$(bindir)/rpm-ostree
+if BUILDOPT_NEW_NAME
+INSTALL_DATA_HOOKS += install-bin-hook
```

Comment:
From the automake manual:

```
Automake generates separate 'install-data' and 'install-exec' rules, in
case the installer is installing on multiple machines that share
directory structure--these targets allow the machine-independent parts to
be installed only once.  
```

But I feel confident in saying basically no one does that.  Even the people who try it are going to be using binary rpms/debs/images, since not everything in the world uses automake.